code/datums/diseases/transformation.dm

```diff
@@ -152,7 +151,6 @@
 
 /datum/disease/transformation/jungle_fever/cure()
 	remove_monkey(affected_mob.mind)
-	GLOB.huds[DATA_HUD_MEDICAL_ADVANCED].remove_hud_from(affected_mob)
```

Comment:
You still need this to remove if they become not a monkey

Reply:
I don't think that's the case. Since the HUD is added after the transformation, and curing does not revert the transformation, I don't think a mob will ever have a HUD at this point, given by the disease.

IIRC cure can only trigger before the mob is transformed, and before they receive the HUD.